Pipelines/Templates/Package.yaml

```diff
@@ -12,6 +12,10 @@ jobs:
     demands: Unity2020LTS
   variables:
     unityLocation: ''
+    ${{ if eq(variables['Build.SourceBranchName'], variables['SigningBranch']) }}:
+      PackageVersion: $(ProductVersion)
+    ${{ if ne(variables['Build.SourceBranchName'], variables['SigningBranch']) }}:
+      PackageVersion: '$(ProductVersion)-prerelease.$(Build.SourceBranchName).$(Build.BuildNumber)'

```

Comment:
Can you set PackageVersion to the -prerelease version first and then override PackageVersion with the release value only if SigningBranch is set?  That way you don't need to maintain two opposite if clauses.

Reply:
Yeah, that would be cleaner.